tex/gloss-russian.ldf

```diff
@@ -216,39 +223,46 @@
       \space \number\year\space г.}%
 }
 
-% The following is based on some ideas from ruscor.sty
+% Russian needs trailing dots in all headings
+\def\xpg@save@autodot{}
+\ifdef{\KOMAScript}{%
+    \let\xpg@save@autodot\autodot%
+}
+
 \def\russian@capsformat{%
+  \ifrussian@forceheadingpunctuation%
    \ifdef{\KOMAScript}{%
-      \ifdef{\chapterformat}{%
-        \renewcommand{\chapterformat}{\prechapter\thechapter\postchapter}}{}%
-      \ifdef{\sectionformat}{%
-        \renewcommand{\sectionformat}{\presection\thesection\postsection}}{}%
-      \ifdef{\subsectionformat}{%
-        \renewcommand{\subsectionformat}{\presubsection\thesubsection\postsubsection}}{}%
-      \ifdef{\subsubsectionformat}{%
-        \renewcommand{\subsubsectionformat}{\presubsubsection\thesubsubsection\postsubsubsection}}{}%
-      \ifdef{\paragraphformat}{%
-        \renewcommand{\paragraphformat}{\preparagraph\theparagraph\postparagraph}}{}%
-      \ifdef{\subparagraphformat}{%
-        \renewcommand{\subparagraphformat}{\presubparagraph\thesubparagraph\postsubparagraph}}{}%
+      \renewcommand*\autodot{.}%
```

Comment:
I think this causes https://github.com/reutenauer/polyglossia/issues/444#issuecomment-706688641
Because `\KOMAScript` can be defined but `\autodot` not (and thus cannot be renewed).

Reply:
Thanks, fixed at 252a03a3debd